backend/src/src-moderation/src/main/java/com/siemens/sw360/moderation/db/ModerationDatabaseHandler.java

```diff
@@ -325,9 +323,13 @@ private void sendMailToModeratorsForUpdateRequest(Set<String> moderators){
         mailUtil.sendMail(moderators,MailConstants.SUBJECT_FOR_UPDATE_MODERATION_REQUEST,MailConstants.TEXT_FOR_UPDATE_MODERATION_REQUEST);
     }
 
-    private void sendMailToUserForDeclinedRequest(String userEmail){
+    private void sendMailToUserForDeclinedRequest(String userEmail, boolean userRequest){
```

Comment:
I've thought about this while writing, but decided against it as I judged the check to be readable well enough.